src/chrome/content/rules/OPM.gov.xml

```diff
@@ -116,15 +115,18 @@
 	<!--	Complications:
 				-->
 	<target host=    "www.leadership.opm.gov" />
+	
+	<!-- Unable to connect via both HTTP and HTTPS
 	<target host=    "servicesonline.opm.gov" />
+	-->
 
 
 	<securecookie host="." name="." />
 
 
-	<!-- could not resolve host -->
+	<!-- could not resolve host 
 	<rule from="^http://servicesonline\.opm\.gov/"
-		to="https://www.servicesonline.opm.gov/" />
+		to="https://www.servicesonline.opm.gov/" />-->
```

Comment:
This is one of those  weird cases where the response code is altered by the server based on the User-Agent:

```shell
legind@EFF-420 ~/workspace/https-everywhere (nextPrime-rm_whitehouse) $ curl -sD /dev/stdout -o /dev/null 'https://www.servicesonline.opm.gov/' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36' | head -n 1
HTTP/1.1 200 OK
legind@EFF-420 ~/workspace/https-everywhere (nextPrime-rm_whitehouse) $ curl -sD /dev/stdout -o /dev/null 'https://www.servicesonline.opm.gov/' | head -n 1HTTP/1.1 403 Forbidden
```

I say leave this unchanged, we can merge anyway if this is the only problematic fetch url.